### PR TITLE
Updating figures in the tutorials

### PR DESCRIPTION
**Pull request by diazlcarlos98**
In this branch I have been updating the first and second tutorials so that the `Visualizer` class' functions are being called properly, as well as replacing the old figures with new ones. Additionally, given that qubit and ancilla indices have changed, I have been carefully checking that the references to these indices in the text actually correspond to the ones in the visualization.

While I'm at it, I've been double-checking to see if I could see any typos or mistakes.

### CODE REVIEW

File: docs/qec_tutorials/decoding/decoding.rst [hunk 1]
```diff
@@ -540,32 +524,36 @@ distributions of each type of error.
 ...                       y=error_probabilities[i, 1],
 ...                       z=error_probabilities[i, 2]) for i in planar_5.data_qubit_indices}
 ... }
->>> x_prob, z_prob = get_effective_probabilities(planar_5, random_qed)
->>> vis_5x = Visualizer(planar_5, error_data=x_prob)
->>> vis_5z = Visualizer(planar_5, error_data=z_prob)
+>>> vis_5 = Visualizer(planar_5, error_data=random_qed)
 
-We visualize the effective :math:`X` error probability distribution:
+We first visualize the $ X $ error probability distribution:
```
Comment (massarom) [inline docs/qec_tutorials/decoding/decoding.rst:529]: Are you sure this is the correct marker for in-line math? Everywhere else `:math:` is used.

**Review by massarom — CHANGES_REQUESTED** [review]
@diazlcarlos98 thanks a lot for this! Can you confirm that if you install `plaquette-graph` from master that tests that is currently failing in the Actions would pass?

**Comment by diazlcarlos98** [comment]
I tried but it did nothing. I don't think that my local installation of `plaquette-graph` would have anything to do with how Git is running the tests. By reading a little bit on the error details, I guess the error would be that somewhere in the C++ binding file, the weights are being passed as a numpy array, instead of the required list of floats.

**Comment by massarom** [comment]
>  I don't think that my local installation of plaquette-graph would have anything to do with how Git is running the tests.

The point is that if _you_ have `graph` installed from its `master` branch and the tests run, this tells us that the problems in the tests is only that the GH Actions are using an old version of `plaquette-graph`. The binding error you see is because the C++ implementation of UF is compiled against a different binary version of `plaquette-graph`, which is not the one on PyPI (from where currently the tests take `plaquette-graph`).

> I tried but it did nothing

Meaning that locally all the tests pass?

**Comment by diazlcarlos98** [comment]
> The point is that if you have graph installed from its master branch and the tests run, this tells us that the problems in the tests is only that the GH Actions are using an old version of plaquette-graph.

Apologies. Yes, that makes sense.

> Meaning that locally all the tests pass?

I just tried running the .py file that was failing (tests/unittests/test_decoders.py) and all of them passed!

**Comment by massarom** [comment]
@diazlcarlos98 there was a fix in the base branch that likely impacts your work here. Can you double-check that all still is consistent after merging the base branch in this PR?

**Comment by diazlcarlos98** [comment]
I'll have to re-do a couple of pictures because of the -previously- missing $Y$ correction in the visualizer.

**Review by massarom — APPROVED** [review]
Thank you @diazlcarlos98! Awesome job!

**Pull request merged**